app/views/ReportsView/index.jsx

```diff
@@ -30,13 +30,10 @@ function ReportsTableComponent(props) {
     gridApi.current = params.api;
     columnApi.current = params.columnApi;
 
-    const opts = {
-      all: true,
-      states: 'ready,active,uploaded,signedoff,archived,reviewed',
-    };
+    const opts = {};
 
-    if (adminUser) {
-      opts.states = 'ready,active,uploaded,signedoff,archived,reviewed,nonproduction';
+    if (!adminUser) {
+      opts.states = 'ready,active,uploaded,signedoff,archived,reviewed';
```

Comment:
If no states are specified does it return them all?

Reply:
Not specifying any states returns all reports regardless of state.